### ISSUE TITLE
Bug in conllu reader when same group closes twice in a line

### ISSUE THREAD

**Issue by amir-zeldes** [gucorpling/spannotator#2]
@yilunzhu I think I found a bug in the conllu reader - can you test GUM_bio_chao? I think this line triggers a bug because the same nested entity group closes twice, which is valid (this is not a data error):

```CoNLL-U
42	family	family	NOUN	NN	Number=Sing	38	nmod	38:nmod:of	Entity=106)105)105)|SpaceAfter=No|XML=</hi>
```
https://github.com/amir-zeldes/gum/blob/master/dep/GUM_bio_chao.conllu#L632

Notice the double closing `105)`.

BTW I also had to flip the entity type and ID columns, since the format is now `(1-person-...` (ID before entity type), but that's easy to do.

**Comment by yilunzhu** [comment]
Will check this out.

**Comment by yilunzhu** [comment]
It is fixed here #3.
